Dockerfile

```diff
@@ -1,5 +1,8 @@
 # syntax=docker/dockerfile:1.7
 
+# This Dockerfile is designed for production deployment, not local development.
```

Comment:
We probably need a similar warning in the `docker-compose.yml` file?

Reply:
Probably - added.